Detectors/data/README.md

```diff
@@ -0,0 +1,35 @@
+# LDMX Detectors
+This directory holds the various LDMX detector descriptions we've used.
+We follow a naming convention to simply stay organized and help with
+collaboration.
+
+## Naming Convention
+`ldmx-<category>-v<version>[-<beam>]`
+
+- All detectors have a `ldmx-` prefix
+- The next string defines the "category" of the detector, generally
+  thought of as structurally different detectors.
+  - The `det` category is the full LDMX detector
+  - The category can have hyphens as well.
+- The string starting with `-v` is the version number for that detector
+- Optionally, a suffix can define the beam energy the detector was
+  designed for. This is optional since some detectors (e.g. the one
+  used for the test beam) do not change their design when the beam
+  energy changes.
+  - If no beam energy suffix is provided, the beam energy is assumed
+    to either be 4GeV or that the detector would not change structure
+    with a different beam.
+
+### Categories
+This defines what the different category names mean so that future
+collaborators understand what the detector is meant to describe.
+- `det`: the full LDMX detector
+- `ti-target`: the full detector but using a Ti target instead of W
```

Comment:
maybe nitpicky but why this different naming convention (including 'target' vs not) between Ti and LYSO targets?

Reply:
not nitpicky at all, a strict and understandable naming convention can be very helpful, I'll remove the `-target` since I think it is overkill.